### DISCUSSION TITLE
Is there a possibility to show loader when snippet is running?

### DISCUSSION THREAD

**Discussion by antosha417** [michaelb/sniprun#178] [Q&A] [answered]
Sometimes my snippets can run for some time doing network calls. And  I don't get any visual feedback that they started to run. 
Do you have any ideas on how can I get an indication that the code started executing?

**Comment by michaelb** [comment, marked as answer]
At the moment, no such feature is supported 😓

Feel free to open a feature request (not sure when I'll have the time to work on that though), and it may not be super useful but enabling "no_output" on your display mode of choice can help you distinguish "still running" and "finished but no output" cases
